documentation/modules/snip-migrating-luns.adoc

```diff
@@ -0,0 +1,10 @@
+:_content-type: SNIPPET
+
+[NOTE]
+====
+{project-short} supports direct migration of logical units numbers (LUNs) from {rhv-full} ({rhv-short}) source providers as follows:
```

Comment:
this sounds like we can list the direct LUNs, like virtual machines, on the source provider and migrate them to the OCP cluster
how about: 
```suggestion
{project-short} supports migration of virtual machines with logical units numbers (LUNs), other than Fibre Channel, from {rhv-full} ({rhv-short}) source providers as follows:
```